src/cli_plugin/index.js

```diff
@@ -0,0 +1,5 @@
+// load the babel options seperately so that they can modify the process.env
+// before calling babel/register
+const babelOptions = require('../optimize/babelOptions').node;
```

Comment:
I think all of these `require` statements should be handled with `imports` - unless there's some technical reason for it.